tidy3d/web/webapi.py

```diff
@@ -677,6 +674,27 @@ def delete_old(
     return len(tasks)
 
 
+@wait_for_connection
+def abort(task_id: TaskId) -> TaskInfo:
+    """Abort server-side data associated with task.
```

Comment:
Maybe "Abort the solver execution if task is currently running."

Reply:
Updated CHANGELOG